pom.xml

```diff
@@ -1707,12 +1707,12 @@ under the License.
 							<rules>
 								<bannedDependencies>
 									<excludes>
-										<exclude>org.yaml:snakeyaml:(,1.30]</exclude>
+										<exclude>org.yaml:snakeyaml:(,1.31]</exclude>
 									</excludes>
 									<includes>
 										<!-- Snakeyaml is pulled in by many modules without using it in production,
 											so there's no benefit in us investing time into bumping these. -->
-										<include>org.yaml:snakeyaml:(,1.30]:*:test</include>
+										<include>org.yaml:snakeyaml:(,1.31]:*:test</include>
```

Comment:
I think in case of movement to 1.33 these two lines should also be updated like `org.yaml:snakeyaml:(,1.32]`

Reply:
I thought about it and decided not to, because snakeyaml 1.32 does not contain a CVE, while everything < 1.32 does. Since this specifically checks for vulnerable snakeyamls, I left this as-is.

Reply:
thanks for clarification